tools/chapel-py/setup.py

```diff
@@ -69,6 +69,10 @@
     chpl_lib_path,
 ]
 
+if str(chpl_variables.get("CHPL_SANITIZE")) == "address":
+    CXXFLAGS += ["-fsanitize=address"]
+    LDFLAGS += ["-fsanitize=address"]
+
```

Comment:
this does not work for `CHPL_HOST_COMPILER=clang`, you need `-shared-libasan` as well